cmds/dump.c

```diff
@@ -7,6 +7,7 @@
 
 #include "libtraceevent/event-parse.h"
 #include "libtraceevent/kbuffer.h"
+#include "mermaid.h"
```

Comment:
I'd like to move the header in the utils directory.

Reply:
I think it doesn't have to be a header. Please see my comments below.